modules/ROOT/pages/create-inbound-message-flow.adoc

```diff
@@ -1,40 +1,50 @@
+To receive messages sent from a trading partner through a 3rd party connection, select an endpoint owned by that connection. 
 
 === Prerequisites
 Before you create your first inbound message flow, you must configure your xref:setup-payload-storage-API.adoc[payload storage API].
 
 === Steps
 
-. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the navigation menu.
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
 . Click *New message flow > Receive from partner*.
-. Either select an existing partner from the list and click *Select* or select *New* to xref:configure-partner.adoc#create-partner[create a trading partner]. +
-A new message flow is created. +
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
++
+. <<receiving-endpoint,Configure the receiving endpoint>>.
+. <<source-message-type,Configure the source message type>>.
+. <<import-map,Import the translation map>>.
+. <<verify-message-flow,Verify that the message flow is complete>>.
+
+[receiving-endpoint]
+== Specify the Receiving Endpoint
+
+Specify the endpoint that will receive the message you are sending. You can do either of the following:
 
-== Configure the Receiving Endpoint
+* Select an existing endpoint owned by the host, target partner, or 3rd party connection.
++
+If you select an endpoint owned by a 3rd party connection, the inbound message goes through that connection.
++
+* Create a new receiving endpoint for this message flow.
+
+To specify the receiving endpoint:
 
 . In the message flow, expand the *Receiving from <partner>* section and click *Select endpoint*.
 . In the *Receiving endpoint* section, click *Select*.
-. In the *Select an endpoint: Receive from <partner>* window:
-.. To use an existing receiving endpoint:
-... Select the endpoint from the list and click *Select*.  +
+. In the *Select an endpoint: Receive from <partner>* window, do either of the following:
+* Select an existing receiving endpoint and click  *Select*.
+* Click *New* to create a new endpoint, as described in xref:<<create-endpoint.adoc, Create and Configure Endpoints>>.
++
 The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
-... Click *Add* to add a unique identifier for the endpoint.
-.. To create a new receiving endpoint:
-... Click *New* next to the search box.
-... From the *Owner* drop-down, select the endpoint's owner.
-... From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
-* xref:endpoint-as2-receive.adoc[AS2]
-* xref:endpoint-ftp-receive.adoc[FTP]
-* xref:endpoint-sftp-receive-target.adoc[SFTP]
 +
-... Click *Save*. +
-The endpoint is added to the message flow.
+* Click *Add* to add a unique identifier for the endpoint.
 
```

Comment:
Please add the new section below (allowing selection of 3rd party AS2 Sender).
https://docs.google.com/document/d/1K__eyTI2TYpE9HGGYzCMfXpx1mNKf99H9s_ss90o7gA/edit#heading=h.nx6vp3px5yrc

**AS2 Sender**

- If the protocol of the Receive endpoint is AS2, select the AS2 Sender from the dropdown menu:
- If your organization will be receiving the AS2 messages directly from the partner for whom the message flow is created, you can select the partner’s name (Default).
- If your organization will be receiving the AS2 messages for the partner via a 3rd party connection such as a VAN or 3PL, you can select the name of the 3rd party connection. The AS2 identifier and certificate of the selected 3rd party connection will be used in AS2 message validation.
- If you selected AS2 as the Receive to partner endpoint protocol, you will need to import the AS2 certificate of the partner or the 3rd party connection that will be sending the AS2 message, if a certificate already does not exist in the partner or 3rd party connection’s profile.

Reply:
I guess this is a special case under Specify the Receiving Endpoint section? For now I am just going to put it in, but want to circle back to it next week.